addon/components/sl-grid.js

```diff
@@ -620,7 +634,7 @@ export default Ember.Component.extend({
      * @returns {undefined}
      */
     disableContinuousPaging() {
-        this.$( '.list-pane .content' ).unbind( 'scroll' );
+        this.$( '.list-pane .content' ).off( 'scroll.sl-grid' );
```

Comment:
No, because it's already specific to this component via this.$
The window listener needed the element id since multiple grids could add resize listeners to the window.